flight/Modules/Actuator/actuator.c

```diff
@@ -532,10 +426,24 @@ static float scaleChannel(float value, float max, float min, float neutral)
 	return valueScaled;
 }
 
+static float channelFailsafeValue(int idx)
+{
+	switch (GetMixerType(idx)) {
+		case MIXERSETTINGS_MIXER1TYPE_MOTOR:
+			return actuatorSettings.ChannelMin[idx];
+		case MIXERSETTINGS_MIXER1TYPE_SERVO:
+			return actuatorSettings.ChannelNeutral[idx];
+		default:
+			// TODO: is this actually right/safe?
+			return 0;
+	}
+
+}
+
 /**
  * Set actuator output to the neutral values (failsafe)
  */
-static void setFailsafe(const ActuatorSettingsData * actuatorSettings, const MixerSettingsData * mixerSettings)
+static void setFailsafe()
```

Comment:
A short wrapper makes it testable that way.  I could plumb everything into the actual calls but I fear it makes the code harder to read from being so verbose.  Upside is it would move even more of the memory consumption from heap.